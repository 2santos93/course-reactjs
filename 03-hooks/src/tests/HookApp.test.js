import { HookApp } from "../components/HookApp"
import {shallow} from 'enzyme';
import '@testing-library/jest-dom'

describe('HookApp testing', () => {
  test('should render ok', () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  })
  
})
