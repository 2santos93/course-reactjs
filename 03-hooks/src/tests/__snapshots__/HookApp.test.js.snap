// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp testing should render ok 1`] = `
<Fragment>
  <h1>
    Hello World!
  </h1>
</Fragment>
`;
